### PR TITLE
Fix Uphill Battle

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

**Comment by dracontes** [comment]
The [[Double Major]] case now works as expected with the token copy entering untapped. 
I did however also want to consider [[Garth One-Eye]] which creates a copy of a card that is then cast. In that case, choosing Shivan Dragon, the resulting token entered untapped, which, as the result of a cast spell, probably shouldn't be the case (see appended screenshot with game log).
image

File: forge-game/src/main/java/forge/game/ability/effects/PlayEffect.java [hunk 1]
```diff
@@ -169,6 +169,7 @@ public void resolve(final SpellAbility sa) {
             // so it gets added to stack
             card.setCopiedPermanent(card);
             card.setToken(true);
+            card.setZone(controller.getZone(ZoneType.None));
```
Comment (Jetz72) [inline forge-game/src/main/java/forge/game/ability/effects/PlayEffect.java:172]: Would it work to just cut out `card.setCopiedPermanent` and `card.setToken` and replace them with `card.setCopiedSpell(true)`? I think PlayEffect is the only place where a card is made into a token while on the stack.

Comment (tool4ever) [reply forge-game/src/main/java/forge/game/ability/effects/PlayEffect.java:172]: yea I'm not super happy with how these params are overlapping, investigating a good path for cleanup might be a worthy project
but just as a note if you didn't notice, I did have to fix this area for that last time:
https://github.com/Card-Forge/forge/pull/5230/commits/b05f9f1

**Review by dracontes — APPROVED** [review]
The Garth!Shivan Dragon now enters tapped. Ran a few more tests with cast from exile effects, like foretell, plot, and plain static command zone effects like Dream-Thief's Bandana to be on the safe side. Also ran Panglacial Wurm through its paces. All of it works as expected 👍

**Pull request merged**